back/routes/adventure.js

```diff
@@ -24,6 +24,33 @@ async function getGoogleRestaurants(radius, lat, lng) {
   return results;
 }
 
+// ----- format to match google api values -----
+function formatRestaurant(restaurant) {
```

Comment:
Great comment and function. This helps me see what Google's API expects and also why we have this function. Maybe one improvement is to update the function name as well - then we might not need the comment at all.
```suggestion
function formatRestaurantForGoogleAPI(restaurant) {
```